frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -68,6 +68,8 @@ export const StyledStreamlitMarkdown =
           // Images in markdown should never be wider
           // than the content area.
           maxWidth: "100%",
+          // In labels, widgets should never be taller than the text.
+          maxHeight: isLabel ? "1em" : "auto",
```

Comment:
I think `maxHeight` auto might not actually exist? If you just want to have a fallback, maybe just set it to undefined here.